### PR TITLE
[CRIMAPP-992] Add partner content to review

### PR DESCRIPTION
**Pull request by hiboabd**
## Description of change 
Introduces attribute `is_included_in_means_assessment` to the partner to facilitate display of the correct content subject in review depending on whether the partner was part of the means assessment 

## Link to relevant ticket
[CRIMAPP-992](https://dsdmoj.atlassian.net/browse/CRIMAPP-992)

## How to manually test the feature

[CRIMAPP-992]: https://dsdmoj.atlassian.net/browse/CRIMAPP-992?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: app/serializers/submission_serializer/definitions/partner.rb [hunk 1]
```diff
@@ -29,9 +29,19 @@ def to_builder
...
       end
       # rubocop:enable Metrics/AbcSize, Metrics/MethodLength
+
+      def partner_in_means_assessment?
+        return false if partner_detail.blank?
+        return true if partner_detail.involvement_in_case == PartnerInvolvementType::NONE.to_s
+        return false unless partner_detail.involvement_in_case == PartnerInvolvementType::CODEFENDANT.to_s
+
+        partner_detail.conflict_of_interest == 'no'
+      end
```
Comment (hiboabd) [inline app/serializers/submission_serializer/definitions/partner.rb:44]: Is there a way to include the type of means assessment module in this class?

File: app/serializers/submission_serializer/definitions/partner.rb [hunk 2]
```diff
@@ -29,9 +29,19 @@ def to_builder
           json.involvement_in_case partner_detail.involvement_in_case
           json.conflict_of_interest partner_detail.conflict_of_interest
           json.has_same_address_as_client partner_detail.has_same_address_as_client
+
+          json.is_included_in_means_assessment partner_in_means_assessment?
```
Comment (hiboabd) [inline app/serializers/submission_serializer/definitions/partner.rb:33]: Is there a way to include the type of means assessment module in this class? I'm getting an error
`undefined local variable or method `crime_application'...`

File: config/locales/en/summary.yml [hunk 3]
```diff
@@ -793,7 +793,7 @@ en:
 
       # START trust_fund
       will_benefit_from_trust_fund:
-        question: Stands to benefit from a trust fund?
+        question: Trust fund
```
Comment (hiboabd) [inline config/locales/en/summary.yml:796]: Noticed the client and partner wording was different

File: app/serializers/submission_serializer/definitions/partner.rb (hunk 2 above)
Comment (arthurashman) [reply app/serializers/submission_serializer/definitions/partner.rb:33]: This was the same issue I had when I added the module to a decision tree https://github.com/ministryofjustice/laa-apply-for-criminal-legal-aid/pull/917/files#r1635155390

Just define crime_application for the module to use @hiboabd

File: config/locales/en/summary.yml (hunk 3 above)
Comment (arthurashman) [inline config/locales/en/summary.yml:796]: ty

**Review by arthurashman — APPROVED** [review]
LGTM. See if the serializer works with def crime_application. If not, it's fine as it

**Pull request merged**